pkg/kubectl/cmd/version.go

```diff
@@ -74,20 +75,25 @@ func RunVersion(f cmdutil.Factory, out io.Writer, cmd *cobra.Command) error {
 		vo.ServerVersion = serverVersion
 	}
 
+	if cmdutil.GetFlagBool(cmd, "short") {
```

Comment:
I saw that discussion but it's too long. I didn't go through the detail. What I see the result of specifying both `--short` and `--output` is the full version. So my reason of the change here is ignore `--output` might be better for user experience. However, I don't have strong opinions on this. If there was enough reason in your former discussion, I'm open to leave the logic as is. Thanks for reviewing, Alejandro !